### DISCUSSION TITLE
Error when opening Ubuntu Terminal

### DISCUSSION THREAD

**Discussion by BenJarrett** [nss-evening-cohort-14/evening-client-side#259] [Help Me!] [answered]
Ben Jarrett

I have not found any other tickets about this error. 

Just went to open my terminal, get the error "[error 0x8007010b when launching `wsl.exe -d Ubuntu-18.04']
Could not access starting directory "//wsl$/Ubuntu-18.04/home/bclja"

https://github.com/microsoft/WSL/issues/3352
https://wikifixes.com/en/errors/0x/0x8007010b/?gclid=Cj0KCQjwhr2FBhDbARIsACjwLo1lazligb485BmEfzRyX45wT7xZdsSrlwbqidq3GcDXFdiR2u1At2YaAhNIEALw_wcB
https://github.com/microsoft/WSL/issues
https://windowstechies.com/-/en/filename-gen/?file=Cmd%20Exe&gclid=Cj0KCQjwhr2FBhDbARIsACjwLo3ZU68uG6PeQmnX4HUkIHZ9GlPuq2ulcuYtuVoSlPEG1gs_9AnNA_gaAhhLEALw_wcB

I cannot type or do anything in the terminal. 
There is no way for me to troubleshoot since I can't type in it. 
Will my files become corrupted if I uninstall and reinstall Ubuntu? 

Not a code issue, no repo to link.

**Comment by drteresavasquez** [comment]
Use bash if you can. Did you update your computer?

**Reply by BenJarrett** [reply]
Haven’t updated my computer at all.
I was working on my personal bio, turned off my computer properly before taking a break to eat, came back to open up my code and this was the error I saw.

Are there instructions for bash setup in the class setup instructions we got at the beginning of class?

**Reply by BenJarrett** [reply]
I used https://itsfoss.com/install-bash-on-windows/ as a tutorial. 

Got everything to work, opened up my project through the terminal.
Ran "npm start"
Got these errors ": not foundram Files/nodejs/npm: 3: 
: not foundram Files/nodejs/npm: 5: 
/mnt/c/Program Files/nodejs/npm: 6: Syntax error: word unexpected (expecting "in")"

Did I miss something with the install?

**Comment by drteresavasquez** [comment, marked as answer]
I'm assuming you fixed this since you have another ticket up. Close this one out please if that is the case by letting us know your solution.

**Reply by BenJarrett** [reply]
Sorry I forgot to reply to this bc I got too caught up in finishing the bio project! 

This is still an issue. 
I was able to work on the project by navigating to it within my VS code.
But I am still getting that error when I open and try to run a project in the bash terminal that I installed

**Reply by drteresavasquez** [reply]
Does the integrated terminal in VSCode work? How are you able to run your application?

**Reply by BenJarrett** [reply]
It does! Looks like I’m going to have to solely use that until I can Get this fixed.
Can I use that like I would a normal terminal to create projects and what not? 

The only way I can run my app is by opening it through Vs code atm

**Reply by drteresavasquez** [reply]
Just go with that for now. Reach out to @JuanDavila1101 and @HunterJuneau   they had similar issues recently.

**Reply by BenJarrett** [reply]
Okay, will do. 

I am trying to clone a repo into my vs code terminal and I am getting 
``` 
Cloning into 'FrontEnd-Capstone'...
git@github.com: Permission denied (publickey).
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
```

How do I get these access rights through the VS Code terminal?
